cmake/SynfigIntltool.cmake

```diff
@@ -1,26 +1,38 @@
 find_program(INTLTOOL_MERGE intltool-merge)
 
 function(SYNFIG_INTLTOOL_MERGE)
```

Comment:
Perhaps it is better to rename it to something with `STUDIO` in the name, so that it indicates that this function can only be used for a studio project?